include/swift/Runtime/Config.h

```diff
@@ -38,7 +38,7 @@
 #ifndef SWIFT_USE_SWIFTCALL
 // Clang doesn't support mangling functions with the swiftcall attribute
 // on Windows and crashes during compilation: http://bugs.llvm.org/show_bug.cgi?id=32000
-#if (__has_attribute(swiftcall) || defined(__linux__)) && !defined(_WIN32)
+#if __has_attribute(swiftcall) && !defined(_WIN32)
```

Comment:
I know they're included in both, and some of the definitions clearly need to be shared; but using feature-detection of the host compiler -- as when this file is included into IRGen and then used to decide whether to emit swiftcall -- is definitely wrong, and it was being "worked around" here by forcing it on "on linux" because the linux host compiler (clang 3.8 which doesn't have swiftcall) answered the has_attribute(swiftcall) probe with no.